packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,931 @@
+
+## Org-wide CAPTCHA settings operations
+
+The Org-wide CAPTCHA settings API has the following operations:
+
+* [Get org-wide CAPTCHA settings](#get-org-wide-captcha-settings)
+* [Update org-wide CAPTCHA settings](#update-org-wide-captcha-settings)
+* [Delete org-wide CAPTCHA settings](#delete-org-wide-captcha-settings)
+
+### Get org-wide CAPTCHA settings
+
+<ApiOperation method="get" url="/api/v1/org/captcha" />
+
+Fetch org-wide CAPTCHA settings.
+
+> **Note:**  If there's no org-wide CAPTCHA settings in current org, it will return an empty org-wide CAPTCHA settings.
+
+#### Request path parameters
+
+N/A
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+N/A
+
+#### Response body
+
+The requested [Org-wide CAPTCHA Setting response](#org-wide-captcha-settings-response-object)
+
+#### Use example
+
+This example returns org-wide CAPTCHA settings:
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/org/captcha"
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "enabledPages": [
+    "SSR",
+    "SSPR",
+    "SIGN_IN"
+  ],
+  "captchaId": "cap18c2Ey3iR9BLDb0g4",
+  "_links": {
+    "captcha": {
+      "href": "http://testorg.okta1.com:1802/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+      "hints": {
+        "allow": [
+          "GET",
+          "POST",
+          "PUT",
+          "DELETE"
+        ]
+      }
+    },
+    "self": {
+      "href": "http://testorg.okta1.com:1802/api/v1/org/captcha",
+      "hints": {
+        "allow": [
+          "GET",
+          "PUT"
+        ]
+      }
+    }
+  }
+}
+```
+
+This example returns an empty org-wide CAPTCHA settings:
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/org/captcha"
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "enabledPages": [],
+  "captchaId": null,
+  "_links": {
+    "self": {
+      "href": "http://testorg.okta1.com:1802/api/v1/org/captcha",
+      "hints": {
+        "allow": [
+          "GET",
+          "PUT"
+        ]
+      }
+    }
+  }
+}
+```
+
+### Update org-wide CAPTCHA settings
+
+<ApiOperation method="put" url="/api/v1/org/captcha" />
+
+Update org-wide CAPTCHA settings.
+
+> **Note:**  `captchaId` cannot be null if `enabledPages` is null or empty.
+
+#### Request path parameters
+
+N/A
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+The [Org-wide CAPTCHA Settings](#org-wide-captcha-settings-object) to update.
+
+#### Response body
+
+The updated [Org-wide CAPTCHA Settings response](#org-wide-captcha-settings-response-object)
+
+#### Use example
+
+This example updates org-wide CAPTCHA settings:
```

Comment:
It would be helpful to describe the nature of this update. It looks like it's enabling the CAPTCHA on the Self-service recovery and sign in pages?